src/commands/docs.rs

```diff
@@ -0,0 +1,66 @@
+//! `docs` subcommand
+
+use abscissa_core::{status_err, Application, Command, Runnable, Shutdown};
+use anyhow::Result;
+
+use crate::{
+    application::constants::{RUSTIC_CONFIG_DOCS_URL, RUSTIC_DEV_DOCS_URL, RUSTIC_DOCS_URL},
+    RUSTIC_APP,
+};
+
+/// Opens the documentation in the default browser.
+///
+/// # Note
+///
+/// If no flag is set, the user documentation will be opened.
+/// If the `dev` flag is set, the development documentation will be opened.
+/// If the `config` flag is set, the configuration documentation will be opened.
+#[derive(Clone, Command, Default, Debug, clap::Parser)]
+#[clap(group = clap::ArgGroup::new("documentation").multiple(false))]
+pub struct DocsCmd {
+    /// Open the user documentation
+    #[clap(short, long, group = "documentation")]
+    user: bool,
+
+    /// Open the development documentation
+    #[clap(short, long, group = "documentation")]
+    dev: bool,
+
+    /// Open the config documentation
+    #[clap(short, long, group = "documentation")]
+    config: bool,
```

Comment:
I think we should not use flags `--user` etc here but just an optional argument such that it can be called as `rustic docs user` or `rustic docs dev`.

Reply:
Yes, fixed it! 👍🏽